### ISSUE TITLE
master deployment: configure_git_user: "/bin/sh: 1: Syntax error: "(" unexpected"

### ISSUE THREAD

**Issue by gavanderhoorn** [ros-infrastructure/buildfarm_deployment#173]
Working around #171 and #172, I run into the following at (I believe) almost the end of the puppet run:

```
/Stage[main]/Profile::Jenkins::Master/Jenkins::Cli::Exec[configure_git_user]/Exec[configure_git_user]/returns (notice): /bin/sh: 1: Syntax error: "(" unexpected
Puppet (err): /usr/bin/java -jar /usr/share/jenkins/jenkins-cli.jar -s http://127.0.0.1:8080 groovy /usr/share/jenkins/puppet_helper.groovy def gitscm_config = Jenkins.getInstance().getDescriptor("hudson.plugins.git.GitSCM"); gitscm_config.setCreateAccountBasedOnEmail(false); gitscm_config.setGlobalConfigName("jenkins"); gitscm_config.setGlobalConfigEmail("jenkins@build.ros.org"); gitscm_config.save(); returned 2 instead of one of [0]
/Stage[main]/Profile::Jenkins::Master/Jenkins::Cli::Exec[configure_git_user]/Exec[configure_git_user]/returns (err): change from notrun to 0 failed: /usr/bin/java -jar /usr/share/jenkins/jenkins-cli.jar -s http://127.0.0.1:8080 groovy /usr/share/jenkins/puppet_helper.groovy def gitscm_config = Jenkins.getInstance().getDescriptor("hudson.plugins.git.GitSCM"); gitscm_config.setCreateAccountBasedOnEmail(false); gitscm_config.setGlobalConfigName("jenkins"); gitscm_config.setGlobalConfigEmail("jenkins@build.ros.org"); gitscm_config.save(); returned 2 instead of one of [0]
```

I haven't spent any time trying to figure out what is going on, just reporting this for now.

`buildfarm_deployment` at 3da95258c079452de52083cc071167ec90bdb15f on a fresh Ubuntu Server 16.04 `amd64` VM.

**Comment by gavanderhoorn** [comment]
Could it be that this:

```
/usr/bin/java -jar /usr/share/jenkins/jenkins-cli.jar -s http://127.0.0.1:8080 groovy /usr/share/jenkins/puppet_helper.groovy def gitscm_config = Jenkins.getInstance().getDescriptor("hudson.plugins.git.GitSCM"); gitscm_config.setCreateAccountBasedOnEmail(false); gitscm_config.setGlobalConfigName("jenkins"); gitscm_config.setGlobalConfigEmail("jenkins@build.ros.org"); gitscm_config.save();
```

is being passed to `sh` in its entirey, instead of the groovy lines being interpreted by the Jenkins cli?

**Comment by nuclearsandwich** [comment]
Resolved by #183
